### PR TITLE
Update Workflow.ru.adoc

### PR DESCRIPTION
**Pull request by likhobory**

### CODE REVIEW

File: content/user/advanced-modules/Workflow.adoc [hunk 1]
```diff
@@ -51,7 +51,7 @@ restricted by the *Run on* property and by the Workflow's conditions.
 |================================================================
 |New Records | The Workflow will run only for new records being created.
 |Modified Records | The Workflow will run only for records that were already created and are being changed.
-|All Records | 
+|All Records a|
```
Comment (samus-aran) [inline content/user/advanced-modules/Workflow.adoc:54]: What's with the a? :thinking:

File: Gemfile.lock [hunk 2]
```diff
@@ -10,4 +10,4 @@ DEPENDENCIES
   asciidoctor (~> 2.0, >= 2.0.10)
 
 BUNDLED WITH
-   1.17.2
+   1.17.2
```
Comment (samus-aran) [inline Gemfile.lock:13]: I assume this was added automatically? Strange one.

Comment (likhobory) [reply Gemfile.lock:13]: Probably yes.

File: content/user/advanced-modules/Workflow.adoc (hunk 1 above)
Comment (likhobory) [reply content/user/advanced-modules/Workflow.adoc:54]: This adds an AsciiDoc style to the table cell and nothing more.

Comment (likhobory) [reply content/user/advanced-modules/Workflow.adoc:54]: For example, without this feature:
\- For *On Save* workflows, this means the Workflow will run for both new and modified records. 
\- For *On Scheduler* Workflows, this means the Workflow will go through every record in the module, 

With this feature:
* For *On Save* workflows, this means the Workflow will run for both new and modified records. 
* For *On Scheduler* Workflows, this means the Workflow will go through every record in the module,

Comment (samus-aran) [reply content/user/advanced-modules/Workflow.adoc:54]: Ahh I see it.
Screenshot_2020-09-08 Workflows

**Pull request merged**